js_modules/dagit/packages/ui/src/components/Group.stories.tsx

```diff
@@ -108,10 +108,7 @@ export const EmptyChildren = () => {
 export const PointerEventsTest = () => {
   return (
     <div>
-      <ButtonLink
-        color={{link: Colors.Blue500, hover: Colors.Yellow500}}
-        onClick={() => console.log('Clicked!')}
-      >
```

Comment:
Undo?

Reply:
Oops, I was cleaning up console.log's and thought I added this one